Dungeoneer/src/com/interrupt/dungeoneer/entities/Text.java

```diff
@@ -54,4 +53,13 @@ public void init(Level level, Level.Source source) {
             z += 0.4f; // Match offset to the new one of
         }
     }
+
+    @Override
+    public void onTrigger(Entity instigator, String value) {
+        if (value == null || value.equals("")) {
```

Comment:
Make sure to call `super.onTrigger(instigator, value);` before this check, as trigger propagation happens in the base class there:
https://github.com/Interrupt/delverengine/blob/b0cd55a2258a4ee7d8cea9cbd910ead4afa33f8a/Dungeoneer/src/com/interrupt/dungeoneer/entities/triggers/BasicTrigger.java#L153

Reply:
Nevermind that, the better method to wire up here would be to override `doTriggerEvent`, making sure to call the super of that at the top as well.

The difference here is that doTriggerEvent is called after the trigger delay, onTrigger before.

Reply:
The `doTriggerEvent` only exists on `Trigger` objects.

Reply:
Nevermind, this is fine as is then.